app/controllers/invite_controller.rb

```diff
@@ -6,7 +6,7 @@ def show
   def create
     garden = Garden.find(params[:garden_id])
     CaretakerNotifierMailer.inform(current_user, params[:email], garden).deliver_now
-    flash[:notice] = "Success! You invited a caretaker to watch your gardens."
+    flash[:success] = "Success! You invited a caretaker to watch your gardens."
```

Comment:
Should this flash message be a singular 'garden' since we're allowing one garden at a time?